assets/prototype/domain/shared/predicates/tickets/filters/index.ts

```diff
@@ -0,0 +1,56 @@
+import archivedOnly from './archivedOnly';
+import expiredOnly from './expiredOnly';
+import nextOnSaleOrPendingOnly from './nextOnSaleOrPendingOnly';
+import onSaleAndPending from './onSaleAndPending';
+import onSaleOnly from './onSaleOnly';
+import pendingOnly from './pendingOnly';
+import percentSoldAtOrAbove from './percentSoldAtOrAbove';
+import percentSoldBelow from './percentSoldBelow';
+import soldOutOnly from './soldOutOnly';
+
+export const now = new Date();
```

Comment:
When coupled with UI will check if this isn't lifted too high in the hierarchy.

Reply:
ya I initially thought we should out that somewhere global, like maybe on the config, 
but don't want to have to drill down for something so basic